src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_InversionFaultSystemRuptSet.java

```diff
@@ -89,6 +89,10 @@ protected static void applySlipRateFactor(FaultSystemRupSet rupSet, NZSHM22_Logi
 		}
 
 		RegionSections tvzSections = new RegionSections(rupSet, new NewZealandRegions.NZ_TVZ_GRIDDED()){
+			@Override
```

Comment:
@voj I'm not clear what this is doing?

Reply:
It's defining a new ad-hoc class because `RegionSections` is abstract. This method is the only one that a subclass needs to implement. And because we don't want to serialise this one, we can simply return null.